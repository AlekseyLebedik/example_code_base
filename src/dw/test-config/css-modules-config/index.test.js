import React from 'react';
import { shallow } from 'enzyme';

import TestComponent from './index';

describe('Jest', () => {
  it('successfully create a snapshot with classname', () => {
    expect(shallow(<TestComponent />)).toMatchSnapshot();
  });
});
