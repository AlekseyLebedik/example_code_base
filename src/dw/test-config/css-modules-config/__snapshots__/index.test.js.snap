// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest successfully create a snapshot with classname 1`] = `
<div
  className="container"
>
  <h1
    className="title"
  >
    Title
  </h1>
  <p
    className="content"
  >
    Text content.
  </p>
</div>
`;
